docker/include/security/tlsfaux/client.py

```diff
@@ -0,0 +1,37 @@
+import socket, ssl, pprint, absolute_filepath, sys, time
```

Comment:
looots of stickler comments so I can't really look at the contents of the file...

Reply:
I've fixed most of these but there's still an issue with using print statements.  I can remove just not sure if this is an actual issue or just some default setting that should be updated on the stickler settings.